org.eclipse.scanning.event/src/org/eclipse/scanning/event/AbstractQueueConnection.java

```diff
@@ -355,41 +360,42 @@ public boolean reorder(U bean, String queueName, int amount) throws EventExcepti
 				submitted.add(index+amount, existing);
 				submitted.remove(index+1);
 			}
-			
+
 			Collections.reverse(submitted); // It goes back with the head at 0 and tail at size-1
-			
-			ISubmitter<U> submitter = this instanceof ISubmitter 
-					                ? (ISubmitter<U>)this 
+
+			ISubmitter<U> submitter = this instanceof ISubmitter
+					                ? (ISubmitter<U>)this
 					                : (ISubmitter<U>)eservice.createSubmitter(getUri(), queueName);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unnecessary cast to "ISubmitter". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1905)